File: backend/src/main/java/com/vti/shop/service/StaffService.java
package com.vti.shop.service;

import com.vti.shop.entity.Staff;
import java.util.List;
import java.util.Optional;

public interface StaffService {
    List<Staff> list();
    Staff create(Staff s);
    Optional<Staff> update(Long id, Staff s);
    boolean delete(Long id);
}

File: frontend/src/services/staffService.js
import axios from "axios";
class StaffService {
  async getAll() {
    return await axios.get("https://manager-shop-phone.onrender.com/api/staff");
  }
  async add(staff) {
    return await axios.post("https://manager-shop-phone.onrender.com/api/staff", staff);
  }

  async update(id, staff) {
    return await axios.put(`https://manager-shop-phone.onrender.com/api/staff/${id}`, staff);
  }

  async remove(id) {
    return await axios.delete(`https://manager-shop-phone.onrender.com/api/staff/${id}`);
  }
}

export default new StaffService();
